src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -0,0 +1,26 @@
+package cromwell.engine.db
+
+import java.util.Date
+
+import cromwell.binding.Call
+import cromwell.engine.store.SymbolStoreEntry
+import cromwell.engine.{WorkflowId, WorkflowState}
+
+trait DataAccess {
+  // Will stamp the start_dt column with DateTime.Now.
+  // Will stamp the WorkflowState as Submitted.
+  def createWorkflow(id: WorkflowId, wdlUri: String, symbols: Seq[SymbolStoreEntry]): Unit
```

Comment:
I agree there shouldn't be dupes, not sure I agree `Set`ifying this is the way to deal with the problem.  We should probably discuss this further on Monday.